homeassistant/components/media_player/apple_tv.py

```diff
@@ -183,20 +202,34 @@ def async_get_media_image(self):
     def media_title(self):
         """Title of current playing media."""
         if self._playing is not None:
+            if self.state == STATE_IDLE:
+                return 'Nothing playing'
             title = self._playing.title
             return title if title else "No title"
 
     @property
     def supported_features(self):
         """Flag media player features that are supported."""
+        features = SUPPORT_TURN_ON | SUPPORT_TURN_OFF
         if self._playing is not None:
             if self.state != STATE_IDLE:
-                return SUPPORT_PAUSE | SUPPORT_PLAY | \
+                features |= SUPPORT_PAUSE | SUPPORT_PLAY | \
                     SUPPORT_SEEK | SUPPORT_STOP | \
                     SUPPORT_NEXT_TRACK | SUPPORT_PREVIOUS_TRACK | \
                     SUPPORT_PLAY_MEDIA
             else:
-                return SUPPORT_PLAY_MEDIA
+                features |= SUPPORT_PLAY_MEDIA
```

Comment:
You can move `SUPPORT_PLAY_MEDIA` to be always part of features and return when `_playing is None` or `state == STATE_IDLE`